src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -2,19 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Diagnostics;
 using System.IO;
 using System.Reflection;
 using System.Resources;
 using System.Runtime.ExceptionServices;
 using Xunit;
-using Xunit.NetCore.Extensions;
 
 namespace System.Tests
 {
-    // No appdomain in UWP or CoreRT
-    [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "dotnet/corefx #18718")]
```

Comment:
@jkotas uapaot does not work for me locally, I tried on the current state and after rebasing to newest and I keep getting this failure:

```
System.Private.CoreLib : Gatekeeper error GK0028 : GK0028 Assembly 'System.Private.CoreLib' is a mixed mode assembly containing managed and native code.  Only pure managed assemblies are compatible with .NET Native.
```